### DISCUSSION TITLE
Social cards - How to display `&sup2;`?

### DISCUSSION THREAD

**Discussion by rei-vilo** [squidfunk/mkdocs-material#7543] [Q&A]
One page has the title...

```
# Use the I&sup2;C bus".
```

...and is referenced in `mkdocs.yml` with 

```
        - Use the I&sup2;C bus: Chapter1/Section2.md
```

Both the page and the TOC render HTML code `&sup2;` correctly with `²` in `Use the I²C bus`.

However, the social card displays...

```
Use the I&sup2;C bus
```  

...instead of the expected 

```
Use the I²C bus
```  

* How to fix this? 
 
Thank you!

**Comment by squidfunk** [comment]
Good catch, HTML entities are not being converted back to code points for the social plugin. I think we can pull in `htmlentitydefs` to do the conversion. Care to submit a PR? If not, please create a proper [bug report](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/) with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by rei-vilo** [comment]
Thank you. See https://github.com/squidfunk/mkdocs-material/issues/7544 for the ticket.
